cert-manager/overlays/emea/jerry/secrets/.sops.yaml

```diff
@@ -0,0 +1,5 @@
+creation_rules:
+  - encrypted_regex: "^(users|data|stringData)$"
+    pgp: >-
+      0508677DD04952D06A943D5B4DC4116D360E3276,
+      2EB7924D7DA3E2DF4717D3EA9E8653F7D343455A
```

Comment:
Again don't recognize this key. But I am pretty sure why this `.sops` file doesn't exist in any other overlay for cert manager is because we have `.sops` file that applies to the whole of the `apps` repo. Therefore we dont need a specific one, but in this case, if that second pgp key is valid and we want to use that, then we will need to keep this file.

Reply:
cool, I'll remove it